OpenSim/Tools/InverseKinematicsTool.cpp

```diff
@@ -262,6 +262,9 @@ bool InverseKinematicsTool::run()
         else
             modelFromFile = false;
 
+        // although newly loaded model will be finalized
+        // there is no gaurantee that the _model has not been edited/modified
+        _model->finalizeFromProperties();
```

Comment:
If the comment on L265 is correct, then a newly loaded model will already be finalized and it would be more logical for this `finalizeFromProperties()` to be part of the `else` block.